content/en/logs/guide/getting-started-lwl.md

```diff
@@ -0,0 +1,132 @@
+
+Click on the log pattern that you would like to exclude to see a sample of underlying logs.
+
+{{< img src="logs/guide/getting-started-lwl/patterns_context_panel.png" alt="Patterns Context" responsive="true" style="width:100%;">}}
+
+The patterns view is helpful when identifying and filtering noisy patterns. It shows the number of logs matching a pattern, split by service and status. Click on the first pattern to view a detailed log of events relating to your status. A contextual panel will populate with information about your noisiest status pattern.
+
+## 3. Create a log pattern exclusion filter
+
+The pattern context panel lists every instance (event) of a log pattern and creates a custom search query based on the selected pattern. Use this query in an exclusion filter to remove those logs from your index.
+
+**To create an exclusion filter**:
+
+1. Click on a pattern from the pattern view list.
+2. Click the **View All** button in the top right corner to automatically generate the search query associated with this pattern.
+3. Select the `</>` option to the right of the search query and copy the search query.
+4. Navigate to the **Configuration** page under Logs in the main menu. Select **indexes** and click on your associated index. This will populate the option to add an exclusion filter.
+5. Select **Add an Exclusion Filter**.
+6. Input the filter name, define the exclusion query by pasting the search query you copied, and set an exclusion percentage.
+
+{{< img src="logs/guide/getting-started-lwl/exclusion_filter.gif" alt="Exclusion Filter" responsive="true" style="width:100%;">}}
+
+**Note**: If a log matches several exclusion filters, only the first exclusion filter rule is applied. A log is not sampled or excluded multiple times by different exclusion filters.
+
+In this example, the service status `info` pattern `Updating recommendations with customer_id=* & url=shops/*/*` is filtered with an exclusion filter. Removing any high volume logging pattern similar to this one from Log Explorer will help you drill down and identify issues quicker. However, these logs are **only** removed from the Log Explorer view. They are still ingested, indexed, and available to view in [Live Tail][6], sent to [log archives][7], or used to [generate metrics][8].
+
+{{< img src="logs/guide/getting-started-lwl/live_tail.gif" alt="Live Tail" responsive="true" style="width:100%;">}}
+
+Exclusion filters can be disabled at any time by toggling the disable option to the right of the filter. They can also be modified and removed by hovering over the filter and selecting the edit or delete option.
+
+## 4. Generate metrics for excluded logs
```

Comment:
Since this section includes the monitor part, we could rename the title to:

```suggestion
## 4. Generate metrics to track excluded logs
```

Note: We should update the other anchors to match this new title